chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -203,6 +204,17 @@ def usage
           self.owner
         end
 
+        def multipath?(device = @device)
+          uuid_path = "/sys/block/#{device}/dm/uuid"
+          File.exist?(uuid_path) && File.open(uuid_path) { |f| f.read(7).start_with?("mpath-") }
+        end
+
+        def multipath_path?
```

Comment:
Thanks @bhdn for this useful information but it will get buried in github never to be found again. That's why I asked for a comment to be embedded in the code to clarify it.